grails-app/controllers/au/org/ala/ecodata/ActivityController.groovy

```diff
@@ -194,6 +194,7 @@ class ActivityController {
             if (isOwner) {
                 render([userIsOwner: isOwner] as JSON)
             } else {
+                response.status = 404
```

Comment:
I actually meant why would we return a 404 if the user is not the owner? This implementation is confusing:
If user is the owner, return [userIsOwner: true]
If user is not the owner, throw a 404 error.

It should always return [userIsOwner: true|false].